content/administration/upgrade.rst

```diff
@@ -1,264 +1,340 @@
+      The database manager displays all databases associated with the user's account. Databases
+      not on the most recent version of Odoo display an arrow in a circle icon next to their name,
+      indicating that they can be upgraded.
 
-.. important::
-   A test database is only intended for testing and remains completely unrelated to your present or
-   future production database. Any data you add, or changes you make, will not be reflected in your
-   upgraded production database.
+      .. image:: upgrade/databases-page.png
+         :alt: The database manager with an upgrade button next to the name of a database.
 
-.. note::
-   Test databases are neutered and features are disabled to prevent them from having an impact on
-   the production database:
+      Click the **arrow in a circle** icon to start the upgrade process. In the popup, fill in:
+
+      - The **version** of Odoo you want to upgrade to, usually the latest version
+      - The **email** address that should receive the link to the upgraded database
+      - The :guilabel:`Purpose` of the upgrade, which is automatically set to :guilabel:`Test` for
+        your first upgrade request
+
+      .. image:: upgrade/upgrade-popup.png
+         :alt: The "Upgrade your database" popup.
+
+      The :guilabel:`Upgrade in progress` tag is displayed next to the database name until
+      completion. Once the process succeeds, an email containing a link to the upgraded test
+      database is sent to the address provided. The database can also be accessed from the database
+      manager by clicking the dropdown arrow before the database name.
+
+      .. image:: upgrade/access-upgraded-db.png
+         :alt: Clicking the menu arrow displays the upgraded test database.
 
-   #. The serial number of the database is modified (to prevent it from sending information as if it
-      was the production database).
-   #. The :ref:`base URL of the database <domain-name/web-base-url>` is reset to
-      ``http://localhost:8069`` and the email domain to ``localhost``.
-   #. Scheduled actions are disabled (the calendar synchronization, the bank statement
-      synchronization, the planned automated actions, the fetching of incoming mail servers, etc.).
-   #. Outgoing mail servers are disabled by archiving the existing ones and adding a
-      fake/non-working one.
-   #. Payment providers and delivery carriers are reset to test environment.
-   #. Accounting localization Electronic Data Interchange (EDI) services are disabled.
-   #. A system parameter is set to tell the database has been neutered.
+   .. group-tab:: Odoo.sh
 
-.. _upgrade/test-db-request:
+      Odoo.sh is integrated with the upgrade platform to simplify the upgrade process.
 
-Request a test database
-=======================
+      .. note::
+         The :guilabel:`Upgrade` tab is available for valid projects with a valid production build
+         in the branches view.
```

Comment:
Part of the current documentation : https://www.odoo.com/documentation/16.0/administration/upgrade/odoo_sh.html#overview

I think it refers to the branch being up and running

Reply:
IMO, it was meant for:
"valid projects": projects with a valid subscription (that are still paying), so for example no trial subscriptions, or educational ones (don't know if they exist on SH)
and "valid production build": what you said.

But it's still seems confusing.

I think we could ask STS (he developed the feature and knows the conditions he added in the code)

Hi @sts-odoo 
Could you help us understand what you meant when you wrote this note in the documentation?
![image](https://github.com/odoo/documentation/assets/28193899/8272b8f5-5757-4a36-bfd0-2eea1213f39e)
What does "valid project" and "valid production build" means?

Reply:
@augustoPerez @nmarotte I confirm that it means a valid project (running  odoo.sh subsrciption on odoo.com not a trial one, ie partnership), AND a valid subcription for the database (running based on odoo.com status).

In other places in the UI of Odoo.sh customer will see if their database is valid or not and why so it is clear enough. The documentation should be complementary to the UI not a copy or replacement of the messages that exist already in the UI.

@augustoPerez does the upgrade platform accepts non valid database to upgrade? If so, why, and do we want that as well on odoo.sh?

Reply:
Hey @sts-odoo , indeed the upgrade platform only accepts database with a valid subscription number, so no subscription, no request. I changed the note to ```The :guilabel:`Upgrade` tab is available for projects with a valid subscription.´´´ since, indeed, if the production branch is red/killed/dropped, it would make no sense to upgrade it. Thanks for your input :D